opencolorio_config_aces/config/cg/generate/config.py

```diff
@@ -353,11 +357,35 @@ def generate_config_cg(
     validate=True,
     describe=ColorspaceDescriptionStyle.SHORT_UNION,
     config_mapping_file_path=PATH_TRANSFORMS_MAPPING_FILE_CG,
+    scheme="Modern 1",
     additional_data=False,
 ):
     """
     Generate the *ACES* Computer Graphics (CG) *OpenColorIO* config.
 
+    The default process is as follows:
+
+    -   The *ACES* CG *OpenColorIO* config generator invokes the *aces-dev*
+        reference implementation *OpenColorIO* config generator via the
+        :func:`opencolorio_config_aces.generate_config_aces` definition and the
+        default reference config mapping file.
+    -   The *ACES* CG *OpenColorIO* config generator filters and and extend
```

Comment:
"and and extend" --> "and extends"

Reply:
Thanks! Little known but powerful feature:

![image](https://user-images.githubusercontent.com/99779/180734586-03a76ef5-beff-48d1-be86-a7d97914ebc3.png)

```suggestion
    -   The *ACES* CG *OpenColorIO* config generator filters and and extends
```